### PR TITLE
blog: flyPadOS3 released

### PR DESCRIPTION
**Pull request by Valastiri**
## Summary of changes
New NOTAM to support release of flyPadOS 3 into development branch.

TODO:

- [x] Featured Image
- [ ] Final Copy Review

## Screenshots(if applicable)
< Provide screenshots of the affected areas if applicable to support your changes being made ensuring you show the full scope of the area changed and surrounding content, this allows for faster reviews on PRs. >

### CODE REVIEW

File: src/posts/flypados3.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+### Support for Over 30 Different Languages
+
+We are extremely happy to offer support for over 30+ languages onboard our EFB. A first in flight simulation, you can now access all the features mentioned above and interact with them in a language of your choice! This would not have been possible without the help of over 200 contributors, and those who have offered to review every corner of the EFB for accuracy. A huge benefit of an open source project is our unique position to have our community assist in endeavors such as this. We cannot stress enough how thankful we are.
+
+[Pictures Here]
+
+### Integrated Pushback
+
+Our new pushback model is smoother with fine control for speed and direction. You can easily utilize your keyboard or various controller axis to steer the tug as you pushback.
```
Comment (frankkopp) [inline src/posts/flypados3.md:51]:
```suggestion
Our new pushback model is smoother with fine control for speed and direction. You can easily utilize your controller axis to steer the tug as you pushback. 
```

No keyboard

File: src/posts/flypados3.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+Our new presets' page allows you to save up to 8 custom presets, name them, and load them up instantaneously at any point during your flight.
+
+![Lighting Preset](/img/notam-images/flypados3/flypad-presets-lighting.png)
+
+#### Aircraft Presets
+
+We heard you liked having an AI co-pilot. This new feature gives your co-pilot control over "on-the-ground" actions up to your takeoff roll. 
+
+Our virtual co-pilot will sequentially performs the necessary setup tasks as in real life from ADIRs alignment to engine starts. Do note that even this feature is realistically
```
Comment (frankkopp) [inline src/posts/flypados3.md:126]:
```suggestion
Our virtual co-pilot will sequentially perform the necessary setup tasks as in real life from ADIRs alignment to engine starts. Do note that even this feature is realistically 
```

**Review by frankkopp — CHANGES_REQUESTED** [review]

File: src/posts/flypados3.md (hunk 1 above)
Comment (dirtyformal) [reply src/posts/flypados3.md:51]: Fixed in a PR submitted into Val's fork.

File: src/posts/flypados3.md (hunk 2 above)
Comment (dirtyformal) [reply src/posts/flypados3.md:126]: Fixed in a PR submitted into Val's fork.

**Review by frankkopp — APPROVED** [review]

File: src/posts/flypados3.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+[Landing Performance Calculator Documentation](https://docs.flybywiresim.com/fbw-a32nx/feature-guides/flypados3/performance/#ground-speed-and-wind).
+
+![Landing OK](/img/notam-images/flypados3/performance-landing-ok.png)
+![Landing MAX](/img/notam-images/flypados3/performance-landing-maxok.png)
+![TOD Distance](/img/notam-images/flypados3/performance-tod-distance.png)
+
+### Interactive "Smart" Checklists
+
+We've included essential interactive checklists, accessible through its own tab in the left menu. The dashboard on the EFB also intelligently recommends the correct checklist based on the current phase of flight you are in.
```
Comment (kevinkiloechovictor) [inline src/posts/flypados3.md:71]:
```suggestion
We have included essential interactive checklists, accessible through its own tab on the left menu. The dashboard on the EFB also intelligently recommends the correct checklist based on the current phase of flight you are in. 
```
Eliminate contraction

File: src/posts/flypados3.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+### Support for Over 30 Different Languages
+
+We are extremely happy to offer support for over 30 languages onboard our EFB. A first in flight simulation, you can now access all the features mentioned above and interact with them in a language of your choice! This would not have been possible without the help of over 200 contributors, and those who have offered to review every corner of the EFB for accuracy. A huge benefit of an open source project is our unique position to have our community assist in endeavors such as this. We cannot stress enough how thankful we are.
+
+![Languages](/img/notam-images/flypados3/flypad-languages.png)
+
+### Integrated Pushback
+
+Our new pushback model is smoother with finer control for speed and direction. You can easily utilize your controller axes to steer the tug as you pushback. 
+We've made it easier to see your turn radius and protected path with the new onboard map. This gives our pilots accurate and precise turns during pushback without having to
```
Comment (kevinkiloechovictor) [inline src/posts/flypados3.md:52]:
```suggestion
We have made it easier to see your turn radius and protected path with the new onboard map. This gives our pilots accurate and precise turns during pushback without having to 
```
Eliminate contraction

File: src/posts/flypados3.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+![TOD Distance](/img/notam-images/flypados3/performance-tod-distance.png)
+
+### Interactive "Smart" Checklists
+
+We've included essential interactive checklists, accessible through its own tab in the left menu. The dashboard on the EFB also intelligently recommends the correct checklist based on the current phase of flight you are in. 
+
+![Checklist 1](/img/notam-images/flypados3/flypad-checklist-page-partly-filled.png)
+![Checklist 2](/img/notam-images/flypados3/flypad-checklist-page-section-filled.png)
+
+The checklists page will also display what checklist you currently have active, and what checklists you are yet to complete. You can use your mouse to click on each line item, or simply use the "Mark item as complete" button at the bottom of the page. If you would like to utilize your hardware, we have some easy to use LVARs for those of you who can write to those.
```
Comment (kevinkiloechovictor) [inline src/posts/flypados3.md:76]:
```suggestion
The checklists page will also display what checklist you currently have active, and what checklists you are yet to complete. You can use your mouse to click on each line item, or simply use the "Mark item as complete" button at the bottom of the page. If you would like to utilise your hardware, we have some easy to use LVARs for those of you who can write to those.
```
Use British spelling

File: src/posts/flypados3.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+
+### Support for Over 30 Different Languages
+
+We are extremely happy to offer support for over 30 languages onboard our EFB. A first in flight simulation, you can now access all the features mentioned above and interact with them in a language of your choice! This would not have been possible without the help of over 200 contributors, and those who have offered to review every corner of the EFB for accuracy. A huge benefit of an open source project is our unique position to have our community assist in endeavors such as this. We cannot stress enough how thankful we are.
+
+![Languages](/img/notam-images/flypados3/flypad-languages.png)
+
+### Integrated Pushback
+
+Our new pushback model is smoother with finer control for speed and direction. You can easily utilize your controller axes to steer the tug as you pushback.
```
Comment (kevinkiloechovictor) [inline src/posts/flypados3.md:51]:
```suggestion
Our new pushback model is smoother with finer control for speed and direction. You can easily utilise your controller axes to steer the tug as you pushback. 
```
Use British spelling

File: src/posts/flypados3.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+We've included essential interactive checklists, accessible through its own tab in the left menu. The dashboard on the EFB also intelligently recommends the correct checklist based on the current phase of flight you are in. 
+
+![Checklist 1](/img/notam-images/flypados3/flypad-checklist-page-partly-filled.png)
+![Checklist 2](/img/notam-images/flypados3/flypad-checklist-page-section-filled.png)
+
+The checklists page will also display what checklist you currently have active, and what checklists you are yet to complete. You can use your mouse to click on each line item, or simply use the "Mark item as complete" button at the bottom of the page. If you would like to utilize your hardware, we have some easy to use LVARs for those of you who can write to those.
+
+### Onboard Navigraph Charts
+
+Every flight requires meticulous planning. Keep your procedures close to hand, with our easily accessible Navigraph charts integration. This way there's no fumbling around with the MSFS toolbar and overlays - ruining your immersion.
```
Comment (kevinkiloechovictor) [inline src/posts/flypados3.md:80]:
```suggestion
Every flight requires meticulous planning. Keep your procedures close at hand, with our easily accessible Navigraph charts integration. This way there is no fumbling around with the MSFS toolbar and overlays, which may ruin your immersion.
```
Fix wording, eliminate contraction

File: src/posts/flypados3.md [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+
+Our new integration now supports the aircraft position symbol for better visualization of where your aircraft is on a particular chart. Our unified experience means that any chart you are viewing will remain active - even if you navigate away from the charts page. We've also included support for pinning relevant procedure maps, which are conveniently displayed on the flyPad's dashboard.
```
Comment (kevinkiloechovictor) [inline src/posts/flypados3.md:82]:
```suggestion
Our new integration now supports the aircraft position symbol for better visualization of where your aircraft is on a particular chart. Our unified experience means that any chart you are viewing will remain active - even if you navigate away from the charts page. We have also included support for pinning relevant procedure maps, which are conveniently displayed on the flyPad's dashboard.
```
Eliminate contraction

File: src/posts/flypados3.md [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+
+![Online ATC](/img/notam-images/flypados3/flypad-online-atc-vatsim.png)
+
+### Presets
+
+### Customizable Lighting Presets
+
+Do you have that perfect cockpit lighting setup that you spend a few minutes on each flight getting just right? Well now you can save it! We know that starting a flight in a cold and dark state means you have a lot to do before you can get off the ground, so we hope this provides a nice little quality of life feature for our pilots.
+
+Our new presets' page allows you to save up to 8 custom presets, name them, and load them up instantaneously at any point during your flight.
```
Comment (kevinkiloechovictor) [inline src/posts/flypados3.md:110]:
```suggestion
Our new presets page allows you to save up to 8 custom presets, name them, and load them up instantaneously at any point during your flight.
```
Remove unnecessary apostrophe

File: src/posts/flypados3.md [hunk 10]
```diff
@@ -0,0 +1,162 @@
...
+
+![Lighting Preset](/img/notam-images/flypados3/flypad-presets-lighting.png)
+
+#### AI Co-Pilot
+
+We heard you liked having an AI co-pilot. This new feature gives your co-pilot control over "on-the-ground" actions up to your takeoff roll. 
+
+Our virtual co-pilot will sequentially perform the necessary setup tasks as in real life - from ADIRs alignment to engine starts. Do note that even this feature is realistically simulated, so the co-pilot will take the appropriate amount of time and steps to complete the procedures and actions. We would like to remind our pilots flying that it's still your responsibility to ensure the MCDU, FCU, transponder and radios are appropriately configured.
```
Comment (kevinkiloechovictor) [inline src/posts/flypados3.md:118]:
```suggestion
Our virtual co-pilot will sequentially perform the necessary setup tasks as in real life - from ADIRs alignment to engine starts. Do note that even this feature is realistically simulated, so the co-pilot will take the appropriate amount of time and steps to complete the procedures and actions. We would like to remind our pilots flying that it is still your responsibility to ensure the MCDU, FCU, transponder and radios are appropriately configured.
```
Eliminate contraction

**Review by kevinkiloechovictor — CHANGES_REQUESTED** [review]

File: src/posts/flypados3.md [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+
+There are other goodies for our pilots to dive into as well, including updated Navigraph support, a new and improved pushback feature with onboard map, lighting presets, and in-flight checklists!
+
+## New Features
+
+### A Refreshing Unified User Experience
+
+At FlyByWire, we believe in well-designed and integrated user interfaces that deliver a great user experience. Since the introduction of our flyPad to MSFS, we have done away with copying real world operating systems, choosing instead to build a cohesive UI that easily integrates useful features at the tip of your cursor. Starting up the EFB requires a simple click on the screen, where you are greeted with our unique dashboard. From here, you can seamlessly download your simBrief OFP to display all the information required for
+your planned flight. The following important information is on the new dashboard:
```
Comment (2hwk) [inline src/posts/flypados3.md:24]:
```suggestion
FlyPad OS has been overhauled, with new improvements thanks to inputs from the team and community feedback. The following important information is on the new dashboard:
```

File: src/posts/flypados3.md [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+
+- Shortcuts to take you to where you want to go.
+- Flight status and general information including your planned route.
+- METAR information - customizable with your choice of ICAO (displayed raw or with simple symbology).
+- Active and upcoming checklists.
+- Pinned Navigraph charts.
+- Active failure management.
+
+This focus on user experience is present throughout our flyPadOS, enabling our pilots to quickly peruse their OFP, access calculations/charts, and integrate easily with online networks such as VATSIM or IVAO.
```
Comment (2hwk) [inline src/posts/flypados3.md:33]:
```suggestion
This new focus on user experience is present throughout our flyPadOS, enabling our pilots to quickly peruse their OFP, access calculations/charts, seamlessly download simBrief OFP, and integrate easily with online networks such as VATSIM or IVAO.
```

File: src/posts/flypados3.md [hunk 13]
```diff
@@ -0,0 +1,160 @@
...
+  
+## Overview
+
+We are happy to introduce the most feature rich update yet to our onboard Electronic Flight Bag (EFB) - flyPadOS 3. The user interface has been completely overhauled with FlyByWire's new design language and principles to improve your experience. Our flyPad is the first EFB to offer support for multiple languages, which would not have been possible without the kind support from our community.
+
+There are other goodies for our pilots to dive into as well, including updated Navigraph support, a new and improved pushback feature with onboard map, lighting presets, and in-flight checklists!
+
+## New Features
+
+### A Refreshing Unified User Experience
```
Comment (2hwk) [inline src/posts/flypados3.md:21]:
```suggestion
### A Refreshed User Experience
```

**Review by 2hwk — CHANGES_REQUESTED** [review]

**Review by 2hwk — COMMENTED** [review]
Other than what I've set out above LGTM and good work

File: src/posts/flypados3.md [hunk 14]
```diff
@@ -0,0 +1,160 @@
...
+  - 'Valastiri'
+  - 'DirtyFormal'
+category: 'UPDATE'
+metaImage: '/img/notam-images/flypados3/flypad-hero.png'
+metaAlt: 'FlyByWire Simulations'
+---
+  
+## Overview
+
+We are happy to introduce the most feature rich update yet to our onboard Electronic Flight Bag (EFB) - flyPadOS 3. The user interface has been completely overhauled with FlyByWire's new design language and principles to improve your experience. Our flyPad is the first EFB to offer support for multiple languages, which would not have been possible without the kind support from our community.
```
Comment (ErickSharp) [inline src/posts/flypados3.md:15]:
```suggestion
After lots of exploration, testing, and development, we are very happy to to announce the release of our most feature rich update yet to our onboard Electronic Flight Bag (EFB) - flyPadOS 3. We have transformed the user interface in an effort to provide a more user-friendly, efficient, and consistent experience when using the flyPad. Along with the strides made in this new iteration of the flyPad, we are proud to say that with the help of over 200 contributors, we are the first to develop an EFB that offers support for multiple languages - enabling flight simmers from across the globe to have a more comfortable and familiar experience when flying.
```

Comment (Valastiri) [reply src/posts/flypados3.md:15]: fixed typo "to to"

File: src/posts/flypados3.md [hunk 15]
```diff
@@ -12,16 +12,13 @@ metaAlt: 'FlyByWire Simulations'
...
 
 ## New Features
 
-### A Refreshing Unified User Experience
+### A Refreshed User Experience
 
-At FlyByWire, we believe in well-designed and integrated user interfaces that deliver a great user experience. Since the introduction of our flyPad to MSFS, we have done away with copying real world operating systems, choosing instead to build a cohesive UI that easily integrates useful features at the tip of your cursor. Starting up the EFB requires a simple click on the screen, where you are greeted with our unique dashboard. From here, you can seamlessly download your simBrief OFP to display all the information required for
-your planned flight. The following important information is on the new dashboard:
+FlyPad OS has been overhauled, with new improvements thanks to inputs from the team and community feedback. The following important information is on the new dashboard:
```
Comment (Benjozork) [inline src/posts/flypados3.md]: flyPadOS, not FlyPad OS

File: src/posts/flypados3.md [hunk 16]
```diff
@@ -0,0 +1,159 @@
...
+### Support for Over 30 Different Languages
+
+We are extremely happy to offer support for over 30 languages onboard our EFB. A first in flight simulation, you can now access all the features mentioned above and interact with them in a language of your choice! This would not have been possible without the help of over 200 contributors, and those who have offered to review every corner of the EFB for accuracy. A huge benefit of an open source project is our unique position to have our community assist in endeavors such as this. We cannot stress enough how thankful we are.
+
+![Languages](/img/notam-images/flypados3/flypad-languages.png)
+
+### Integrated Pushback
+
+Our new pushback model is smoother with finer control for speed and direction. You can easily utilise your controller axes to steer the tug as you pushback.
+We have made it easier to see your turn radius and protected path with the new onboard map. This gives our pilots accurate and precise turns during pushback without having to switch to an external view.
```
Comment (ErickSharp) [inline src/posts/flypados3.md:48]:
```suggestion
We have made it easier to see your turn radius and projected path with the new onboard map. This gives our pilots accurate and precise turns during pushback without having to switch to an external view.
```

**Pull request merged**